src/Invoker.php

```diff
@@ -44,6 +46,12 @@ public function __construct(callable $target, string $signatureType)
             throw new InvalidArgumentException(sprintf(
                 'Invalid signature type: "%s"', $signatureType));
         }
+        $this->errorLogFunc = function (string $error) {
+            fwrite(fopen('php://stderr', 'wb'), json_encode([
```

Comment:
I like that we're flagging the error severity. I wonder if we should use `JSON_UNESCAPED_SLASHES` or `JSON_UNESCAPED_UNICODE`?

Reply:
We can use both via `JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES`

I don't know the implications of this however. Is this what we want?

Reply:
done!